app/javascript/application.js

```diff
@@ -1,18 +1,17 @@
 // Entry point for the build script in your package.json
 
-import 'jquery'
+import 'jquery' // Blacklight 7 needs jQuery. Remove when we upgrade to Blacklight 8?
 import bootstrap from  "bootstrap/dist/js/bootstrap"
 global.bootstrap = bootstrap // Required for Blacklight 7 so it can manage the modals
 
 import 'blacklight-frontend/app/assets/javascripts/blacklight/blacklight'
+import 'simple-datatables'
```

Comment:
comment that this is for reports view??  I don't feel strongly.

Reply:
Added.